zerver/models.py

```diff
@@ -480,6 +480,13 @@ class Realm(models.Model):
         blank=True,
         on_delete=models.SET_NULL,
     )
+    reportmsg_stream: Optional["Stream"] = models.ForeignKey(
```

Comment:
We have a project policy of avoiding abbreviations in names for things -- so we'd always spell out `report_message_stream` or similar. But maybe `abuse_report_stream` would be clearer?

Reply:
DONE - expanded abbreviation from msg => message (simple expansion for now - some of the identifiers are getting gigantic!)